src/Validator/ValidatorFactory.php

```diff
@@ -16,7 +16,7 @@ class ValidatorFactory
     private $defaultTranslator;
     private $constraintValidatorFactory;
 
-    public function __construct(?ValidatorInterface $validator, ConstraintValidatorFactoryInterface $constraintValidatorFactory, ?TranslatorInterface $translator)
+    public function __construct(?ValidatorInterface $validator, ?ConstraintValidatorFactoryInterface $constraintValidatorFactory, ?TranslatorInterface $translator)
     {
         $this->defaultValidator = $validator;
```

Comment:
@murtukov why this is injected it is never used here?

Reply:
@mcg-web I think I just forgot about it. The `InputValidator` uses 2 validators (default and custom). The default one is used to read metadata from classes (required for the `link` feature). Later I wanted to move both of them into ValidatorFactory, thats why I injected the `$defaultValidator` into `ValidatorFactory`, but seems like I forgot about it or it wasn't necessary anymore, I don't remember.

Anyways you can savely remove it from `ValidatorFactory`.